lesson 3. exercise 1.py

```diff
@@ -1,48 +1,47 @@
-def game():
```

Comment:
Now the program works differently, how will you merge the past and the current versions?